docker-compose.yaml

```diff
@@ -106,37 +98,27 @@ services:
       - "15432:5432"
     volumes:
       - ./init/postgres:/docker-entrypoint-initdb.d/
-    deploy:
-      resources:
-        limits:
-          cpus: "1"
-          memory: 500M
 
   mysql:
     image: mysql:8.0
     container_name: playground-mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=mysql
+      - MYSQL_USER=mysql
+      - MYSQL_PASSWORD=mysql
+      - MYSQL_DATABASE=db
     ports:
       - "13306:3306"
     volumes:
       - ./init/mysql:/docker-entrypoint-initdb.d/
-    environment:
-      MYSQL_ROOT_PASSWORD: mysql
-      MYSQL_USER: mysql
-      MYSQL_PASSWORD: mysql
-      MYSQL_DATABASE: db
     command:
       --default-authentication-plugin=mysql_native_password
       --character-set-server=utf8mb4
       --collation-server=utf8mb4_general_ci
       --explicit_defaults_for_timestamp=true
       --lower_case_table_names=1
-    deploy:
-      resources:
-        limits:
-          cpus: "1"
-          memory: 500M
     healthcheck:
-      test: ["CMD-SHELL", "mysqladmin ping -h localhost -p ${MYSQL_ROOT_PASSWORD}"]
+      test: ["CMD-SHELL", "mysqladmin ping -h localhost -pmysql"]
       interval: 5s
       timeout: 60s
       retries: 5
```

Comment:
Maybe needs to add `start_period: 20s` in there?

Reply:
Added.